llvm/utils/TableGen/SubtargetEmitter.cpp

```diff
@@ -1902,8 +1902,9 @@ void SubtargetEmitter::parseFeaturesFunction(raw_ostream &OS) {
     StringRef Instance = R->getName();
     StringRef Value = R->getValueAsString("Value");
     StringRef FieldName = R->getValueAsString("FieldName");
+    bool SetMaxValue = R->getValueAsBit("SetMaxValue");
 
-    if (Value == "true" || Value == "false")
+    if (Value == "true" || Value == "false" || !SetMaxValue)
       OS << "  if (Bits[" << Target << "::" << Instance << "]) " << FieldName
```

Comment:
The generated code here processes each feature in alphabetical order. Its not connected to the order on the command line or the order declare in tablegen. So this doesn't do "last specified value".